### ISSUE TITLE
UI suggestions to make managing planets easier

### ISSUE THREAD

**Issue by PeterMardahl** [freeorion/freeorion#2484]
I'm using version 0.4.5 on Linux/Ubuntu.....

Suggestions:
1)  Pop up menus for specifying builds on planets (right click on planet, menu appears)
2)  Pop up menus for selecting planets in a system (right click on planet, planet menu appears)
3)  Hotkeys for selecting from these pop ups

Here's the UI issue I'm hoping to fix.  In order to queue up a build on a system, I have to click on that system in the galaxy map.  Then sweep the mouse, aim at the planet, and click on it.  Then sweep the mouse across the screen, find the item I want to build, and then click on that.  That's three mouse sweeps that require pretty precise aim.

Here's how it could work:
Right click on system, hotkey-select planet 3, hotkey select the build I want.  That's just one mouse sweep and then keyboard clicks.  If the hotkeys are selected to be either left-handed or right handed, I don't even have to move that hand.  Like mark the planets with hotkeys "abcdefgqwertzxcvs".  The build menu could be similar.

All the mouse sweeping and precise clicking is giving me wrist issues, and this UI change would make a big difference on that!

Thanks for consideration,

--PeterM

**Comment by geoffthemedio** [comment]
>1)  Pop up menus for specifying builds on planets (right click on planet, menu appears)
>2)  Pop up menus for selecting planets in a system (right click on planet, planet menu appears)

The difference between these suggestions is unclear.

**Comment by geoffthemedio** [comment]
>1)  Pop up menus for specifying builds on planets (right click on planet, menu appears)
>2)  Pop up menus for selecting planets in a system (right click on planet, planet menu appears)

The difference between these suggestions is unclear.

**Comment by TheSilentOne1** [comment]
I think what he means is:
2. Pop up menus for selecting planets in a system (right click on **system**, planet menu appears)

**Comment by toolforger** [comment]
General advice: Avoid z for left-hand hotkeys, various keyboard layouts have z and y swappen (and unfortunately that other key is right-handed).
+1 on reducing mouse miles. It's pretty annoying and has been getting worse with increasing screen size.

**Comment by abmpicoli2** [comment]
Yep, there should be an easy way to navigate through all colonies you have. The current arrows on system browser windows for example :
![image](https://user-images.githubusercontent.com/37077317/164334899-f9288428-0515-401b-a66a-060945deca4b.png)
These arrows are quite useless: they go through the systems in alphabetical order, right? So, if I use those arrows, I'm looking at my Dagaz-alpha system I need to click those arrows through tons of "Deep Space" systems to finally reach to my other colony at "Nacre-alpha". 

THe only useful window for managing an empire any bigger than 3 or 4 systems is the objects window: although at first view it is an AWESOME tool, kudos for the idea, to have only one window to view everything would demand at the very least (all INCLUSIVE)
* Options to sort through multiple fields: For example, how can I see what systems have good planets and doesn't have enemy monsters over it? You don't. You will have to sieve through the whole list. 
* Quick views: be able to quick change from one view to another. For example, from a "system suitability view for the Fulver" to a "fleets by system and owner" in a single click. Each view showing their own files.
* Quick filters: be able to, say, when I see a good planet for the Fulver, I can filter through that system to see what else is there : "Oh! bummer, there is a fleet of sentinels at the same system :( "
* Multiple object view windows * : to have only one view to see both planets and fleets only creates noise, that gets bigger and bigger as there are more fleets in the game.

If I want to go through my colonies, I need to open the only window available for *all* objects, unselect all objects but planets and then add a filter for "owned colonies". And then choose me as the target civilization. 

Hm... how are my fleets? Oh, I need to open the object window again. click fleets, unselect planets, and oh, environment for species doesn't make sense for fleets, let's change that field... oh, I need this extra field. 

And now I've lost the view for "owned colonies". Because I only have one object window to work with everything. As long as you have a very , very, very, very small game, that has only , I dk , 20 systems, this is not so much of an issue. As the game grows, however, this becomes a pain.

Also, the Production system should have navigation arrows to go between each system where I can build stuff. (As it is in Master of Orion II).

**Comment by geoffthemedio** [comment]
> Yep, there should be an easy way to navigate through all colonies you have. The current arrows on system browser windows for example [...] are quite useless: they go through the systems in alphabetical order, right? So, if I use those arrows, I'm looking at my Dagaz-alpha system I need to click those arrows through tons of "Deep Space" systems to finally reach to my other colony at "Nacre-alpha".

Note that there are keyboard shortcuts to cycle through all systems or just those with your empire's populated planets:

![keyboard_shortcuts](https://user-images.githubusercontent.com/11386908/164404285-610d3078-8b7e-44b4-8c59-d0aa37222727.png)

**Comment by abmpicoli2** [comment]
> > Yep, there should be an easy way to navigate through all colonies you have. The current arrows on system browser windows for example [...] are quite useless: they go through the systems in alphabetical order, right? So, if I use those arrows, I'm looking at my Dagaz-alpha system I need to click those arrows through tons of "Deep Space" systems to finally reach to my other colony at "Nacre-alpha".
> 
> Note that there are keyboard shortcuts to cycle through all systems or just those with your empire's populated planets:
> 
> ![keyboard_shortcuts](https://user-images.githubusercontent.com/11386908/164404285-610d3078-8b7e-44b4-8c59-d0aa37222727.png)

For sure this is better than nothing. I wasn't aware of those. Yet I still think there should be ways inside the UI to do the exact same thing. You are already committed to the mouse to do everything production related... *except* navigate through systems, and the shortcut demands two hands. And there are still some extra clicks there, right? When I hit Ctrl-comma I go through the planets in the system? Or just the system? What if the system has two planets? 

There should be a simple way to navigate through colonies.
